test/compilable/extra-files/ddoc_markdown_escapes.html

```diff
@@ -0,0 +1,504 @@
+
+<!DOCTYPE html>

```

Comment:
Maybe we should have something like dlang.org's verbatim.ddoc file here too?

https://github.com/dlang/dlang.org/blob/master/verbatim.ddoc

Might making updating and debugging easier.